doc/source/index.rst

```diff
@@ -228,6 +228,7 @@ Papers
    ray-overview/index.rst
    ray-libraries.rst
    installation.rst
+   Ray Job Submission <ray-job-submission/overview.rst>
```

Comment:
<img width="327" alt="Screen Shot 2021-11-18 at 5 09 21 PM" src="https://user-images.githubusercontent.com/5459654/142523561-34fe1627-c5bf-483d-8cd4-69f885aadfe7.png">
Did we already decide this is where it should go?  My gut says this is too prominent for an alpha feature.  Should it go under Ray Core > Using Ray?  Or somewhere else?

Reply:
Good question -- i'm using info i had from @javi-redondo 's discussion https://docs.google.com/document/d/1Vr2my31WOELUNDALR0LgL3Cmp6nVAoLulRp_plNcfZM/edit#heading=h.8729z75wh8kl that structured our logs as 

OSS
- Overview of Ray > Gentle introduction to Ray
- Developing Ray Applications
   - Running Ray locally
   - Connecting to Ray on a cluster
   - Moving to production
     - This is where Job submission should go 
   - Debugging

This feature didn't really belong to existing library and to me better suited as a ray-level dev infra component, so we should think about where to put it now if the new structure is not there yet.

Reply:
Thanks for the clarification, I moved it to the "Ray Core" section between "Configuring Ray" and "Ray Dashboard".